### ISSUE TITLE
MyKadRecognizerResult .ownerBirthDate get nil

### ISSUE THREAD

**Issue by leeyikkong** [BlinkID/blinkid-ios#54]
May I know why I cannot get the birthdate after scan the MyKad?
keep get nil and make the app crashed.

Thank you

**Comment by culoi** [comment]
Hi leeyikkong,

Can you please send us the high quality image of the document you're tying to scan?

BR

**Comment by leeyikkong** [comment]
Hi,

you can try to find MyKad image from google.
After i update to latest version of blinkid, it cause this problem.
Previously Im using version 2.4.0 and do not have this problem.

Thank You

**Comment by leeyikkong** [comment]
Hi,

for iKad, it also got this issue, date of birth = nil

Thank you

**Comment by hollowaykeanho** [comment]
Reproduced the same issue in version `2.5.1` . DoB is always nil despite NRIC is clear and correct.

Current workaround in Swift3:
```
var dict:[String:String] = [:]
let index = data.nricNumber?.index((data.nricNumber?.startIndex)!, offsetBy: 6)
let dateString = data.nricNumber?.substring(to: index!)
let dateFormatter = DateFormatter()
dateFormatter.dateFormat = "YYMMdd"
let date = dateFormatter.date(from: dateString!)
dateFormatter.dateFormat = "'Date of Birth: ' dd MMM YYYY"
dict["birthday"] = dateFormatter.string(from: date!)
```

Please fix.

**Comment by culoi** [comment]
@leeyikkong @hollowaykeanho 
Thank you for reporting the error!
Issue will be fixed in new release which is expected on January 31st. 
Best regards!

**Comment by leeyikkong** [comment]
hi,
when will release the new version?
thank you

**Comment by culoi** [comment]
hi @leeyikkong 

Please email me ivan.culo (at) microblink.com 

I can send you a prerelease version.

Best regards

**Comment by leeyikkong** [comment]
hi @culoi 

May I know where to send the email to you?
contact us or?

thank you

**Comment by culoi** [comment]
Hi @leeyikkong 

I wrote you my email in previous post.

or you can go through contact us 

Best regards

**Comment by leeyikkong** [comment]
(@)

**Comment by culoi** [comment]
Yes, (at) = @

**Comment by leeyikkong** [comment]
ok thank you so much

**Comment by leeyikkong** [comment]
Hi,

may I know if buy the license for the BlinkID, is it all the functionality
in the blinkid can be use? is that any limitation?
thank you

**Comment by culoi** [comment]
Hi @leeyikkong 

For licensing and price detail, please contact our sales team: https://microblink.com/en/products/blinkID

Regards

**Comment by leeyikkong** [comment]
ok thanks

**Comment by Cerovec** [comment]
Hi,
this issue should be fixed in the latest build 2.6.0. If so, please close this issue.
Thanks,
Jurica.

**Comment by leeyikkong** [comment]
will check it in this few days and if no problem then will close it .thanks alot
